mod/01_rating/lib/javascript/drag_and_drop.js.coffee

```diff
@@ -0,0 +1,260 @@
+sortDragItems = (list, key, order='desc') ->
+  order = if order == 'desc' then 1 else -1
+  $(list).find('.yinyang-row').sortElements (a, b) ->
```

Comment:
probably obvious, but we should probably try to update the name "yinyang" at some point, since the yinyang design elements are all gone now...